.kokoro/tests/run_tests.sh

```diff
@@ -100,6 +100,33 @@ for file in **/requirements.txt; do
     nox -s "$RUN_TESTS_SESSION"
     EXIT=$?
 
+    # If this is a continuous build, send the test log to the Build Cop Bot.
+    # See https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop.
+    if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
+      # Use the service account with access to the repo-automation-bots project.
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+      gcloud config set project repo-automation-bots
```

Comment:
Would it be better to use `-project` flag on the command below instead? Hopefully none of the samples are relying on gcloud to detect projects, but it might be safer to minimize side effects

Reply:
Good idea! Switched.